src/Magnum/LibOVRIntegration/Conversion.h

```diff
@@ -0,0 +1,157 @@
+        return {other.x, other.y};
+    }
+
+    static ovrVector2i to(const Vector<2, Int>& other) {
+        return {other[0], other[1]};
+    }
+};
+
+// ovrVector2f
+template<> struct VectorConverter<2, Float, ovrVector2f> {
+    static Vector<2, Float> from(const ovrVector2f& other) {
+        return {other.x, other.y};
+    }
+
+    static ovrVector2f to(const Vector<2, Float>& other) {
+        return {other[0], other[1]};
+    }
+};
+
+// ovrVector3f
+template<> struct VectorConverter<3, Float, ovrVector3f> {
+    static Vector<3, Float> from(const ovrVector3f& other) {
+        return {other.x, other.y, other.z};
+    }
+
+    static ovrVector3f to(const Vector<3, Float>& other) {
+        return {other[0], other[1], other[2]};
+    }
+};
+
+// ovrMatrix4
+template<> struct RectangularMatrixConverter<4, 4, Float, ovrMatrix4f> {
+    static RectangularMatrix<4, 4, Float> from(const ovrMatrix4f& other) {
+        return Matrix4<Float>::from(reinterpret_cast<const Float*>(other.M));
+    }
+
+    static ovrMatrix4f to(const RectangularMatrix<4, 4, Float>& other) {
+        return {other[0][0], other[0][1], other[0][2], other[0][3],
+                other[1][0], other[1][1], other[1][2], other[1][3],
+                other[2][0], other[2][1], other[2][2], other[2][3],
+                other[3][0], other[3][1], other[3][2], other[3][3]};
+    }
+};
+
+// ovrQuatf
+template<> struct QuaternionConverter<Float, ovrQuatf> {
+    static Quaternion<Float> from(const ovrQuatf& other) {
+        return Quaternion<Float>(Vector3<Float>{other.x, other.y, other.z}, other.w);
+    }
+
+    static ovrQuatf to(const Quaternion<Float>& other) {
+        const Vector3<Float> imaginary = other.vector();
+        return {imaginary.x(), imaginary.y(), imaginary.z(), other.scalar()};
+    }
+};
+
+// ovrPosef
+template<> struct DualQuaternionConverter<Float, ovrPosef> {
+    static DualQuaternion<Float> from(const ovrPosef& other) {
+        return DualQuaternion<Float>::translation(Vector3<Float>(other.Position))*DualQuaternion<Float>{Quaternion<Float>(other.Orientation)};
+    }
+
+    static ovrPosef to(const DualQuaternion<Float>& other) {
+        return {ovrQuatf(other.rotation()), ovrVector3f(other.translation())};
+    }
+};
+
+// ovrRecti
+template<> struct RangeConverter<2, Int, ovrRecti> {
+    static Range<2, Int> from(const ovrRecti& other) {
+        return Range<2, Int>::fromSize(Vector2i(other.Pos), Vector2i(other.Size));
+    }
+
+    static ovrRecti to(const Range<2, Int>& other) {
+        return {ovrVector2i(other.min()), ovrSizei(other.size())};
+    }
+};
+
+}} /* namesapce Math::Implementation */
+
+namespace LibOvrIntegration {
+
+// ovrTexture
+/**
+ * @brief Wrap an ovrTexture as @ref Texture2D.
+ * @param texture @ref ovrTexture to wrap.
+ * @return texture as @ref Texture2D.
+ */
+Texture2D wrap(const ovrTexture& texture);
+
+}} /* namespace Magnum::LibOvrIntegration */
+
+#endif /* Magnum_LibOvrIntegration_Conversion_h */
```

Comment:
Actually, I added those :D And, yes, I also had to change it a few times already. In that case I will remove them again.